From 14225d12038086b1859e001b7c9677cc00840af1 Mon Sep 17 00:00:00 2001
From: jaeyeol <a01092201761@gmail.com>
Date: Fri, 12 Jul 2024 15:31:59 +0900
Subject: [PATCH] zzfsdf

---
 jaeyeol1 | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/jaeyeol1 b/jaeyeol1
index 9118d6c..c1c1465 100644
--- a/jaeyeol1
+++ b/jaeyeol1
@@ -1 +1,8 @@
-asdasd
\ No newline at end of file
+develop
+sdafsdafdsafs
+
+ewrwer
+
+
+
+zxzxzxzx
\ No newline at end of file
-- 
2.45.2.windows.1

